src/main/java/com/code_intelligence/jazzer/mutation/api/SerializingMutator.java

```diff
@@ -29,7 +29,7 @@
 @DoNotMock("Use TestSupport#mockMutator instead")
 public abstract class SerializingMutator<T> implements Serializer<T>, ValueMutator<T> {
   @Override
-  public final String toString() {
+  public String toString() {
```

Comment:
This function is `final` since subclasses shouldn't override it. Instead, they should (and have to) override `getDebugString`.